packages/admin-panel/src/pages/resources/editSurvey/Cells.jsx

```diff
@@ -0,0 +1,33 @@
+import { TableCell } from '@material-ui/core';
+import styled from 'styled-components';
+
+export const Cell = styled(TableCell)`
+  padding-inline: 0.2rem;
+  padding-block: 0.4rem;
+  border-color: ${({ theme }) => theme.palette.grey['400']};
+  border-style: solid;
+  border-width: 0 1px 1px 0;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  // TODO: fix this once we apply resizable columns
```

Comment:
Is there a ticket for this?

Reply:
Apologies, I missed this, have removed the comment